distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time.enso

```diff
@@ -506,8 +506,9 @@ type Date_Time
 
        Arguments:
        - amount: The amount of time to add to this instant, either `Duration` for
-         adding time (hours, minutes, etc.), or `Period` for adding date (days,
-         months, years).
+         adding time (hours, minutes, etc.), `Period` for adding date (days,
+         months, years), or `Date_Period` for adding a single unit (year,
+         quarter, month, etc.).
```

Comment:
Should have `Time_Period` as @radeusgd says. Probaly not "either" if 4 options. 
Might be easier to just lose the stuff after this instant.

Reply:
How about just:
```
       - amount: The amount of time to add to this instant, It can be
         `Duration`, `Period`, `Date_Period`, or `Time_Period`.
```